webapp/src/components/channel_header_button/index.js

```diff
@@ -5,13 +5,12 @@ import {connect} from 'react-redux';
 import { bindActionCreators } from 'redux';
 
 import {showRHSPlugin} from 'actions';
-import {isPluginRhsOpen} from 'selectors';
 
 import ChannelHeaderButton from './channel_header_button';
 
 function mapStateToProps(state) {
     return {
-        shouldHighlight: isPluginRhsOpen(state),
+        isVisible: state['plugins-com.mattermost.plugin-todo'].isVisible,
```

Comment:
Can you make a selector for this in the `selectors.js` file? There are other components that are not using the selectors, but I think any new code should use the selectors.